cmd/mutating-webhook-service/pkg/inject/common.go

```diff
@@ -0,0 +1,91 @@
+package inject
```

Comment:
The `cmd/mutating-webhook-service/pkg` directory should be moved out (`internal/pkg` maybe?)